Data Confidentiality in Open Science

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title:  " Data Confidentiality in Open Science"
+date:   2023-11-06
+author: Konstantinos Rigas
+category: practice
+tags: reusability
+---
+
+Data Confidentiality in Open Science
+
+I. INTRODUCTION
+
+Open science represents a groundbreaking approach to research, emphasizing transparency, collaboration, and the sharing of knowledge, data, and methodologies within 
+the scientific community. [1] A central pillar of open science is the availability of research data. Nonetheless, ensuring data confidentiality within the framework of 
```

Comment:
Can you add citations as hyperlinks to the DOI? https://www.markdownguide.org/basic-syntax/#reference-style-links

Reply:
hyperlinks to the DOI have been added